scripts/bin/revbump

```diff
@@ -1,95 +1,125 @@
+
+FOLDERS = list(json.load(open(REPO_PATH)).keys())
+
+
+def is_dep(DEP: str, line: str) -> bool:
+    """
+    Checks if the dep is in line
+    """
+    tmp_deps = line.split("=")[1]
+    if tmp_deps.startswith('"'):
+        deps = tmp_deps.split('"')[1].split(", ")
+    if tmp_deps.startswith("'"):
+        deps = tmp_deps.split("'")[1].split(", ")
+    else:
+        deps = tmp_deps
+    return DEP in deps
+
+
+def get_build_dependent_files(folders: list, DEP: str) -> list:
+    """
+    Gets all the packages that depend on some package
+    """
+    build_files = []
+    for d in folders:
+        for folder in os.listdir(d):
+            for line in (
+                open(os.path.join(d, folder, "build.sh"), "r").read().split("\n")
+            ):
+                if line.startswith("TERMUX_PKG_DEPENDS") or line.startswith(
+                    "TERMUX_PKG_BUILD_DEPENDS"
+                ):
+                    if is_dep(DEP, line):
+                        build_files.append(os.path.join(d, folder, "build.sh"))
+    return build_files
+
+
+def bump_revision(file: str) -> None:
+    """
+    Bumps version in file
+    """
+    opened_file = open(file, "r+")
+    file_read = opened_file.read().split("\n")
+    revision = 0
+
+    if "TERMUX_PKG_REVISION" in "\n".join(file_read):
+        for line in file_read:
+            if line.startswith("TERMUX_PKG_REVISION"):
+                revision = line.split("=")[-1]
+                if revision.startswith('"'):
+                    revision = revision.split('"')[1]
+                revision = int(revision) + 1
+                file_read[file_read.index(line)] = (
+                    line.split("=")[0] + "=" + str(revision)
+                )
+                break
+
+    else:
+        for line in file_read:
+            if line.startswith("TERMUX_PKG_VERSION"):
+                file_read.insert(file_read.index(line) + 1, "TERMUX_PKG_REVISION=1")
+                revision = 1
+                break
+
+    opened_file.write("\n".join(file_read))
+    opened_file.close()
+    print("{} -> {} {}".format(revision - 1, revision, file))
+
+
+def locate_dir(file: str) -> str:
+    """
+    Finds package in repo
+    """
+    for repo in FOLDERS:
+        if file in os.listdir(repo):
+            return f"{repo}/{file}/build.sh"
+    return exit("Package {} not found".format(file))
+
+
+def parse_args() -> None:
+    if len(sys.argv) == 1 or sys.argv[1] in ["--help", "-h"]:
+        print(usage)
+        exit(1)
+
+    if sys.argv[1] in ["--dependencies", "-d"] and len(sys.argv) > 2:
+        build_files = get_build_dependent_files(FOLDERS, sys.argv[2])
+        print("Bumping all {} files".format(len(build_files)), end="\n\n")
+        for files in build_files:
+            bump_revision(files)
+        exit(1)
+
+    if len(sys.argv) >= 2 and sys.argv[1] not in ["--dependencies", "-d"]:
+        print("Bumping {} packages".format(len(sys.argv[1:])))
+        packages = sys.argv[1:]
```

Comment:
Please handle the case if the package is given as an absolute path as well, for example `./bin/revbump packages/openssl`, for example by taking the basename of the arg